docs/src/examples/components/Dropdown/Types/DropdownExampleSearchMultiple.shorthand.steps.ts

```diff
@@ -8,17 +9,16 @@ const selectors = {
     `.${Dropdown.slotClassNames.selectedItems} span:nth-child(${itemIndex})`,
 }
 
-const steps: ScreenerSteps = [
-  (steps, keys) =>
-    steps
-      .click(selectors.toggleIndicator)
-      .click(selectors.item(2))
-      .click(selectors.toggleIndicator)
-      .click(selectors.item(2))
-      .keys(selectors.input, keys.leftArrow)
-      .snapshot('Selects last selected element')
-      .hover(selectors.selectedItem(1))
-      .snapshot('Hovers first selected element'),
-]
+const steps: ScreenerSteps = applyCommonThemesBeforeStep((steps, keys) =>
```

Comment:
I don't like this change 👎 I suggest to add `themes` as handlable export:
```
import { themes } from '@stardust-ui/react'

export const themes = [themes.teamsDark]
// or
export { themes }
```

Reply:
fixed and applied change as discussed

Reply:
fixed and applied change as discussed